pkg/cmd/pulumi/package_add.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024, Pulumi Corporation.
+//Copyright 2024, Pulumi Corporation.
```

Comment:
```suggestion
// Copyright 2024, Pulumi Corporation.
```

Reply:
Honestly I don't know what is changing this line over and over.  I may need to obliterate my editor settings!